### DISCUSSION TITLE
using-slack-workflows-for-better-jira-bug-triaging/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#18] [Comments]
# using-slack-workflows-for-better-jira-bug-triaging/

Like many engineering teams, the Photobox app team lets colleagues from other departments report bugs internally via Slack. To ensure sufficient detail, we use a Slack workflow to standardise bug reporting and simplify Jira ticket raising. Here’s how to implement something similar.

https://blog.jakelee.co.uk/using-slack-workflows-for-better-jira-bug-triaging/

**Comment by conanmoriarty** [comment]
👏🏻👏🏻 v nice!
